pkg/kv/kvserver/kvflowcontrol/kvflowcontrol.go

```diff
@@ -72,10 +72,10 @@ func (m ModeT) String() string {
 // SafeFormat implements the redact.SafeFormatter interface.
 func (m ModeT) SafeFormat(p redact.SafePrinter, verb rune) {
 	if s, ok := modeDict[m]; ok {
-		p.Print(s)
+		p.SafeString(redact.SafeString(s))
```

Comment:
This would be good, but the map is used for things that require `map[ModeT]string`, e.g. [this](https://github.com/cockroachdb/cockroach/blob/3b6f3c659f7c511761b44d65ba2cf62815a514c1/pkg/kv/kvserver/kvflowcontrol/kvflowcontrol.go#L38-L46) cluster setting definition.